GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -2783,6 +2782,12 @@ private void DiffContextMenu_Opening(object sender, System.ComponentModel.Cancel
             }
         }
 
+        protected override void RestorePosition(string name)
+        {
+            base.RestorePosition(name);
```

Comment:
Don't you mind if I change it to work the same way as in FormCommit? FormCommit use event OnLoad to restore splitter position after restoring form size